README.md

```diff
@@ -136,6 +136,29 @@ additional [installation
 instructions](https://github.crookster.org/switching-to-straight.el-from-emacs-26-builtin-package.el/)
 for moving your package management from package.el to straight.
 
+## treesitter
+
+You can try the new native treesitter mode `rust-ts-mode` with:
+
+```elisp
+(use-package rustic
+  :init
```

Comment:
Probably this is more idiomatic:

```
:custom
(rustic-treesitter-derive t)
```

Reply:
We need to set the variable before rustic.el is required. Unfortunately we can't set the parent mode dynamically(at least as far I know) so we have to configure this at initialization.

Reply:
Ah, I see. Thanks!